include/mbedtls/rsa.h

```diff
@@ -110,18 +283,24 @@ typedef struct
 }
 mbedtls_rsa_context;
 
+#else
+
+#include "rsa_alt.h"
+
+#endif /* MBEDTLS_RSA_ALT */
```

Comment:
Could you please use the old model for the preprocessor guards?
- The way you do it is better than how we use it in other modules and consistent with the way we want to do it in the future
- I know, that we did not have clear guidance on how to enable replacement for new modules when you made these changes
- But now we have a clear instruction to stick with the old model for now and switch to the new one later at the same time for all modules

Reply:
@yanesca Sure, if there has been agreement on another model in the meantime, I'll follow that. I will check.